### ISSUE TITLE
ScrollYOffset is always set to 0 when the page is refreshed

### ISSUE THREAD

**Issue by jeff1010322** [rohit-gohri/redocusaurus#162]
ScrollYOffset option seems to work if you navigate to a page already loaded, but if you refresh or navigate directly to an API page the offset gets set to 0.  So the sidebar will scroll under the navbar.  

This affects the examples as well, so you can see what happens by directly navigating here and scrolling: https://redocusaurus.vercel.app/examples/using-single-yaml/

![Screen Shot 2022-04-04 at 8 08 47 AM](https://user-images.githubusercontent.com/2868139/161540575-c0c97d73-5534-4f4a-a041-1cf6d52185c1.png)

It seems that this is issues is related to this line: https://github.com/rohit-gohri/redocusaurus/blob/e81229325921eaa73d60f3914a14570721f583f7/packages/docusaurus-theme-redoc/src/utils/useSpec.ts#L28

It seems this is in place to prevent the warning that appears when using a selector on a direct page load:
```
scrollYOffset value is a selector to non-existing element. Using offset 0 by default
```

I don't know how to preserve the offset if you want to use a selector, but it would at least be nice to keep the value if you set it to something other than a selector.

**Comment by jeff1010322** [comment]
I took a shot at creating a partial fix for this issue.  See the linked PR.  Basically just keeping the logic to disable this option when it is set to a selector.

With this change you could fix the examples by changing the redoc options here to use a number:
https://github.com/rohit-gohri/redocusaurus/blob/main/website/docusaurus.config.js#L50

Like this:
```
      options: { disableSearch: true, scrollYOffset: 60 },
```

I couldn't find a solution to actually allow selectors to load correctly, however.

**Comment by rohit-gohri** [comment]
Merging that fix for now to unblock you, will fix actual issue later

**Comment by rohit-gohri** [comment]
Released in v1.0.2

**Comment by benfoster** [comment]
I faced the same issue and couldn't get a selector to work either e.g. `.navbar`.
